framework/customize

```diff
@@ -139,6 +140,16 @@ _customize_has_remote_repo () {
   return 1
 }
 
+_customize_get_default_branch () {
+  if [ "$CUSTOMIZE_DOTFILES_CUSTOM_DEFAULT_BRANCH" == '' ]; then
+    _customize_ensure_jq
+    _customize_ensure_hub
+    CUSTOMIZE_DOTFILES_CUSTOM_DEFAULT_BRANCH=$(hub api "/repos/$(_customize_github_get_user)/$DOTFILES_CUSTOM_REPO" | jq .default_branch)
```

Comment:
Is the idea here that, when there's no custom branch name specified in the dotfiles config, to use the default branch name of the customization repo?

Reply:
Does this definitely run after that repo gets created?

Reply:
The idea is to use the default branch specified in the remote config, there is a default when you just create a repo before pushing any branches. It worked in my test, though I didn't explore extensively.